public/static/js/version.js

```diff
@@ -7,6 +7,35 @@ mciModule.controller('VersionController', function($scope, $rootScope, $location
   $scope.tab = 0
   $scope.version = {};
   $scope.taskStatuses = {};
+  $scope.subscriptions = [];
+  $scope.triggers = [
+      // TODO uncomment as we implement these
+      // {
+      //    trigger: "failure",
```

Comment:
Please add two more blocks here, one for outcome, one for succcess.

Reply:
Same as for task, let's add once we implement